envoy/api/v2/core/grpc_service.proto

```diff
@@ -53,6 +54,10 @@ message GrpcService {
     //    streams_total, Counter, Total number of streams opened
     //    streams_closed_<gRPC status code>, Counter, Total streams closed with <gRPC status code>
     string stat_prefix = 3 [(validate.rules).string.min_bytes = 1];
+
+    // Additional configuration for site-specific customizations of the Google
+    // gRPC library.
+    google.protobuf.Struct config = 4;
```

Comment:
How is this going to work? Where will the struct be passed to?

Reply:
I have a followup PR for the envoyproxy side which will split out the channel creation into a site-specific file that can be overriden at import. It will be passed the full `GoogleGrpc` proto.